acl_upgrade.php

```diff
@@ -97,6 +97,7 @@
  *       ADD  amendment   Amendments                (Physicians,Clinicians(addonly))
  *       ADD  lab         Lab Results               (Physicians,Clinicians(addonly))
  *       ADD  docs_rm     Documents Delete          (Administrators)
+ *       ADD  pat_rep     Patient Report            (Administrators)
```

Comment:
Should create a 5.0.2 section below in this text and place this entry there

Reply:
Hi @bradymiller .Fixed